doc/lua_api.txt

```diff
@@ -2271,18 +2271,20 @@ Examples
 Version History
 ---------------
 
-* FORMSPEC VERSION 1:
+* Formspec version 1 (old):
```

Comment:
```suggestion
* Formspec version 1 (5.0.0 or older):
```

list 5.0.0 as it is still in the 5.x family

Reply:
... or `prior to 5.1.0` to include `5.0.1`